package.json

```diff
@@ -78,15 +83,16 @@
     "start": "react-native start",
     "start:android": "react-native run-android",
     "start:ios": "react-native run-ios",
-    "lint": "eslint index.js --max-warnings=0",
```

Comment:
why allow no warnings?

Reply:
I don't intent to set policy but in my experience lint is like - either you do it all the way, or you don't
I usually set all my projects like this, then for very specific cases you can lint ignore a specific line in a file, with justification in the associated PR. Too harsh? I dunno, works in practice though

Reply:
I agree, I think it's better to keep the flag as it was before, we can ignore inline